.github/workflows/release.yml

```diff
@@ -88,12 +88,12 @@ jobs:
     name: "Deploy Release: DEV"
     needs: [ build_release, build_frontend_release ]
     if: github.ref == 'refs/heads/dev-rheft'
-    environment: staging
+    environment: dev
```

Comment:
This is a bug where the incorrect environment name was set for the `dev` deployment.